lib/components/inputs/Select/SingleValue/SingleValue.tsx

```diff
@@ -3,18 +3,30 @@ import { components, SingleValueProps } from 'react-select'
 import SpanTooltip from '../../../SpanTooltip'
 
 import './singleValue.scss'
-interface ValueProps extends SingleValueProps{
-    data: any
+
+interface ValueProps extends SingleValueProps {
+  data: {
+    label: string
+    icon?: React.ReactNode
+    subLabel?: string
+  }
 }
-function SingleValue(props:ValueProps) {
-  const { data: { icon, label } } = props
+
+function SingleValue(props: ValueProps) {
+  const {
+    data: { icon, label, subLabel }
+  } = props
```

Comment:
Can be replaced with the above syntax

Reply:
when we have more than 2 props in a component, we try to move them out to a variable to make it more readable